src/inspect_ai/_util/hash.py

```diff
@@ -3,7 +3,7 @@
 
 def mm3_hash(message: str) -> str:
     # Generate the 128-bit hash as two 64-bit integers
-    h1, h2 = mmh3.hash64(message.encode("utf-8"))
+    h1, h2 = mmh3.hash64(message.encode("utf-8"))  # pylint: disable=E0633
```

Comment:
I did dive on this one a little bit. `pylint`, thinks that this function returns `int` rather than `tuple[int, int]`. It's suggested on the net that pylint's type resolution code is not as robust as VS Code's particularly when it comes to _stub_ definitions.